ruler-frontend-tests/src/test/kotlin/com/spotify/ruler/frontend/testutil/WebDriverExtension.kt

```diff
@@ -47,6 +46,7 @@ class WebDriverExtension : BeforeAllCallback, BeforeEachCallback, AfterEachCallb
             .addArguments("--start-maximized")
             .addArguments("--no-sandbox")
             .addArguments("--disable-dev-shm-usage")
+            .addArguments("--remote-allow-origins=*")
```

Comment:
Is this needed? 🤔

Reply:
yeah, fixes some Selenium issues

Reply:
This (and maybe the Selenium dependency upgrades) seem to be a separate issue not related to this PR (i.e. #110 is also failing). Probably should be a separate PR to unblock other PRs while this one is getting reviewed?

Reply:
done